src/docs/markdown/CONFIGURATION.md

```diff
@@ -18,6 +21,25 @@ REACT_APP_STYLE_TYPE=grayscale
 REACT_APP_SHOW_LOGIN_ICON=true
 ```
 
+## At runtime, using `condaStoreConfig`
+
+When using a webpacked version of `conda-store-ui`, you might want to pass it a configuration. 
+In your HTML file, add the following **before** loading the react app :  
+
+```html
+<script>
+    var condaStoreConfig = {
```

Comment:
replace `var` for `const` here!

Reply:
The PR looks good to me. I know my suggestion does not affect the current behavior, but let's use 'const' to clarify that those values do not change.